samples/CookieSample/Startup.cs

```diff
@@ -28,8 +28,8 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
             {
                 if (!context.User.Identities.Any(identity => identity.IsAuthenticated))
                 {
-                    var user = new ClaimsPrincipal(new ClaimsIdentity(new[] { new Claim(ClaimTypes.Name, "bob") }, CookieAuthenticationDefaults.AuthenticationScheme));
-                    await context.Authentication.SignInAsync(CookieAuthenticationDefaults.AuthenticationScheme, user);
+                    var user = new ClaimsPrincipal(new ClaimsIdentity(new[] { new Claim(ClaimTypes.Name, "bob") }, CookieDefaults.AuthenticationScheme));
+                    await context.Authentication.SignInAsync(CookieDefaults.AuthenticationScheme, user);
```

Comment:
We should keep it Authentication if its unclear, so I can add that back for Cookies